be/src/models/dbo.models.ts

```diff
@@ -6,6 +6,7 @@ export interface Vocabulary {
   description: string;
   created: Date;
   lastModified: Date;
+  entityCount?: number;
```

Comment:
In the `openapi.yaml` is the `entityCount` specified as `required`, we have reached a development status where we can remove the `?`, please do so